exercises/svm/ex_rnw/sol_svm_kernel_definition.Rnw

```diff
@@ -0,0 +1,99 @@
+\begin{enumerate}
+	\item 
+
+		The function is symmetric, as we can prove in the following way:
+
+		\begin{equation}
+			\kxxt  =  \xv^T \xtil  = \xtil^T \xv  =  k(\tilde \xv, \xv)
+		\end{equation}
+
+		To check if the kernel gram matrix is positive definite, we will remember the definition of $\xv$
```

Comment:
I guess, capital X would be better here to avoid confusion with lower case x in the line above?

Reply:
Yes! Changed